### ISSUE TITLE
In world crafting

### ISSUE THREAD

**Issue by JasonMcRay** [MineMaarten/PneumaticCraft#811]
Hi. Just a request, would it be possible to have a config option to remove the in world crafting for compressed iron and etching the PCBs for special cases in modpacks? Or maybe use an API function within my own mod? We have got an idea for our Infitech modpack, where we would move the etching to GT machine Chemical Bath.
Thank you

**Comment by MineMaarten** [comment]
Yes, but not for MC1.7.10 if that's what you were targeting, only major bugfixes will be done on the MC1.7.10 branch.
